server/libs/simplelanguage/CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+This repository contains a snapshot of SimpleLanguage that is updated only after major changes. The development version is part of the Truffle repository: https://github.com/graalvm/truffle
```

Comment:
It's very likely better to figure out how to add simplelanguage as a submodule. This way, it would be easier to keep it up to date, by fetching up-stream changes.